hudi-common/src/main/java/org/apache/hudi/common/table/view/RemoteHoodieTableFileSystemView.java

```diff
@@ -147,7 +153,7 @@ public RemoteHoodieTableFileSystemView(String server, int port, HoodieTableMetaC
     String url = builder.toString();
     LOG.info("Sending request : (" + url + ")");
     Response response;
-    int timeout = 1000 * 300; // 5 min timeout
+    int timeout = this.timeoutSecs * 1000; // msec
```

Comment:
Does the Request.connect expect timeout in millisecs ?

Reply:
Yes, msec.